ADRs/RH/SIG-SRE/ADR-00002 Personas related to Managed Services.md

```diff
@@ -0,0 +1,114 @@
+# ADR-00002 Personas related to Managed Services
+
+Authors: Craig Robinson
+
+
+## Status
+
+Draft
+
+
+## Problem Statement
+
+A successful SRE practice requires a cross-functional group of stakeholders to engage and work across teams to focus on reliability. This document lists the Personas involved in supporting a cloud service. For example, as a stakeholder of SLOs, I want to be able to consume the data in a relevant and consistent way in accordance with my "persona". The roles and responsibilities of each Persona will be documented in RACI format in a future PR.
```

Comment:
for the non-RH folks it might be useful to add a footnote on what RACI means

Reply:
Great idea @dastergon . I have just updated and linked that text to https://en.wikipedia.org/wiki/Responsibility_assignment_matrix

Reply:
Once the RACI document is done maybe we can loop back and link to that document here.